crates/forge-runner/src/context_data.rs

```diff
@@ -0,0 +1,14 @@
+use camino::Utf8PathBuf;
+use cheatnet::runtime_extensions::forge_runtime_extension::contracts_data::ContractsData;
+use std::collections::HashMap;
+
+pub struct ContextData {
+    pub runtime_data: RuntimeData,
+    pub workspace_root: Utf8PathBuf,
+    pub test_artifacts_path: Utf8PathBuf,
+}
+
+pub struct RuntimeData {
+    pub contracts_data: ContractsData,
+    pub environment_variables: HashMap<String, String>,
+}
```

Comment:
I don't understand clearly what is semantic distiction between those

Reply:
`ContextData` is a general purpose struct that stores data related to current context of the run (e.g. info about specific paths in the package as u can see). Meanwhile `RuntimeData` is a more specific struct, it stores the data that is needed in the runtime of test execution

Reply:
I can comment, but imo `Runtime` is a better name